www/docs/main/example-apps.md

```diff
@@ -102,6 +103,7 @@ Here's some example apps:
       <td>
         <ul>
           <li><a href="https://githubbox.com/trpc/trpc/tree/main/examples/express-server">CodeSandbox</a></li>
+          <li><a href="https://stackblitz.com/edit/trpc-express-server?file=README.md">StackBlitz</a></li>
```

Comment:
This should be something like

https://stackblitz.com/github/trpc/trpc/tree/main/examples/express-server

Reply:
fixing them